core/src/navigation/LeftNav.html

```diff
@@ -62,33 +62,46 @@
-              class="fd-nested-list__link has-child {isExpanded(nodes, expandedCategories) ? 'is-expanded' : ''}"
-              aria-haspopup="true"
-              aria-expanded="{isExpanded(nodes, expandedCategories)}"
-              on:click|preventDefault="{() => setExpandedState(nodes, !isExpanded(nodes, expandedCategories), this)}"
-            >
-              {#if isOpenUIiconName(nodes.metaInfo.icon)}
-              <span
-                class="fd-nested-list__icon {'sap-icon--' + nodes.metaInfo.icon} {isSemiCollapsed && !nodes.metaInfo.icon ? 'sap-icon--rhombus-milestone-2' : ''}"
-                role="presentation"
-              ></span>
-              {:else}
-              <span class="fd-nested-list__icon">
-                <img
-                  src="{nodes.metaInfo.icon}"
-                  alt="{nodes.metaInfo.altText ? nodes.metaInfo.altText : ''}"
-                >
-              </span>
-              {/if}
-              <span
-                class="fd-nested-list__title"
-              >{$getTranslation(key)}</span>
-            </a>
+            <div class="fd-nested-list__content has-child" tabindex="0">
+              <a
+                href="javascript:void(null)"
+                title="{$getTranslation(key)}"
+                class="fd-nested-list__link {isExpanded(nodes, expandedCategories) ? 'is-expanded' : ''}"
+                tabindex="-1"
+                aria-haspopup="true"
+                aria-expanded="{isExpanded(nodes, expandedCategories)}"
+                on:click|preventDefault="{() => setExpandedState(nodes, !isExpanded(nodes, expandedCategories), this)}"
+              >
+                {#if isOpenUIiconName(nodes.metaInfo.icon)}
+                <span
+                  class="fd-nested-list__icon {'sap-icon--' + nodes.metaInfo.icon} {isSemiCollapsed && !nodes.metaInfo.icon ? 'sap-icon--rhombus-milestone-2' : ''}"
+                  role="presentation"
+                ></span>
+                {:else}
+                <span class="fd-nested-list__icon">
+                  <img
+                    src="{nodes.metaInfo.icon}"
+                    alt="{nodes.metaInfo.altText ? nodes.metaInfo.altText : ''}"
+                  >
+                </span>
+                {/if}
+                <span
+                  class="fd-nested-list__title"
+                >{$getTranslation(key)}</span>
+              </a>
+              <a
+                class="fd-nested-list__expand-icon"
+                href="#"
+                tabindex="-1"
+                aria-controls="EX400L2"
+                aria-haspopup="true"
+                aria-expanded="{isExpanded(nodes, expandedCategories)}"
+                on:click|preventDefault="{() => setExpandedState(nodes, !isExpanded(nodes, expandedCategories), this)}"
+              ></a>
+            </div>
             <ul
               class="fd-nested-list fd-nested-list--text-only level-2"
+              id="EX400L2"
```

Comment:
I have a question regarding `id` and` aria-controls`. For which reason do we have it here? If it's a mechanism to control open/close items, then it's redundant, since we have another mechanism to control it. If it's a part of WCAG, then I would say we need a dynamic `id` and` aria-controls`, otherwise we have the same values ​​on all expanded items.